dew/dependencyprocessor.py

```diff
@@ -100,7 +100,7 @@ def has_dewfile(self) -> bool:
         dewfile_path = os.path.join(self.get_remote().get_source_dir(), 'dewfile.json')
         return os.path.isfile(dewfile_path)
 
-    def get_dewfile(self) -> DewFile or None:
+    def get_dewfile(self) -> Optional[DewFile]:

```

Comment:
Thanks! Still somewhat sour python didn't decide to allow `or` to be used to specify a union of types.